.github/workflows/use-action.yaml

```diff
@@ -0,0 +1,44 @@
+---
+name: use-action
+
+on:
+  push:
+    tags-ignore:
+      - "**"
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**.md'
+
+jobs:
+  use-action:
+    name: use-action
+    runs-on: ubuntu-latest
+    steps:
+      # checking out the project code on the current workspace
+      - uses: actions/checkout@v3
+      # preparing the Kubernetes cluster# (KinD) and installing kubectl
+      - uses: helm/kind-action@v1.2.0
+        with:
+          version: v0.14.0
+          cluster_name: kind
```

Comment:
Should we change this to create it's own cluster? Or are the other repositories already setup to use this cluster name?

Reply:
The workflow in this repository is employed to test the action, to test itself in other words. When using this action in other repositories the cluster must be provided.

Reply:
To be clear, this action does not create the cluster. That's a dependency that must be provided.

Reply:
However, in the workflow, where the action is being tested, then we need to simulate the regular use and provide a cluster. Please consider the [usage example](https://github.com/otaviof/setup/blob/6725064a396cb86431d2fb15b18968b1c04260b7/README.md?plain=1#L29-L30).

Reply:
> To be clear, this action does not create the cluster. That's a dependency that must be provided.

That's fine. My point was: how does your action know the name of the KinD cluster? Problems (if any) might become visible if you change the name here to a non-default value.

Reply:
> That's fine. My point was: how does your action know the name of the KinD cluster? Problems (if any) might become visible if you change the name here to a non-default value.

All it needs is a `kubectl` in the `${PATH}` with sufficient permissions. The cluster name is completely irrelevant, as well as the nature of the cluster installer (in this context KinD).

This thread is happening at the workflow file, while the action itself is defined at the `action.yaml`, please consider the later.

Reply:
> > To be clear, this action does not create the cluster. That's a dependency that must be provided.
> 
> That's fine. My point was: how does your action know the name of the KinD cluster? Problems (if any) might become visible if you change the name here to a non-default value.

@SaschaSchwarze0, @coreydaley, please consider the latest changes, I've created a new input on which we can tweak the KinD cluster name. Thanks for the heads up!